### PR TITLE
Input/Output Scaling Discontinued

### PR DESCRIPTION
**Pull request by kiritigowda**
Scaling of input image and output image using color convert is
discontinued.

Build loomsl dll with the following flags.
Multi-threaded Debug DLL (/MDd)
Multi-threaded DLL (/MD)

### CODE REVIEW

**Comment by rgiduthuri** [comment]
I've committed /MD change in all CMakeLists.txt files in amdovx-core and amdovx-modules.
Please merge this branch to the latest develop branch and adjust the title (multithreaded DLL part is not needed anymore).

**Comment by kiritigowda** [comment]
CMakeList.txt changes removed.

File: vx_loomsl/live_stitch_api.cpp [hunk 1]
```diff
@@ -2471,15 +2437,15 @@ LIVE_STITCH_API_ENTRY vx_status VX_API_CALL lsInitialize(ls_context stitch)
...
 		vx_imagepatch_addressing_t addr_out = { 0 };
 		void *ptr[1] = { nullptr };
-		addr_out.dim_x = (vx_uint32)(stitch->output_rgb_scale_factor * stitch->output_buffer_width);
-		addr_out.dim_y = (vx_uint32)(stitch->output_rgb_scale_factor * stitch->output_buffer_height);
+		addr_out.dim_x = stitch->output_buffer_width;
+		addr_out.dim_y = stitch->output_buffer_height;
 		addr_out.stride_x = 3;
-		addr_out.stride_y = stitch->output_buffer_stride_in_bytes;
+		addr_out.stride_y = stitch->output_buffer_width * 3;
 		if (addr_out.stride_y == 0) addr_out.stride_y = addr_out.stride_x * addr_out.dim_x;
```
Comment (rgiduthuri) [inline vx_loomsl/live_stitch_api.cpp]: This can be problematic. Did I miss something?

Comment (kiritigowda) [inline vx_loomsl/live_stitch_api.cpp]: ChromaKey input image is always a RGB image and its y stride will be width times three, this change also fixes the bug; if output image is not RGB, then chroma input image will have problems.

**Pull request merged**